frontend/elements/src/core/buttons/arrow-button.ts

```diff
@@ -0,0 +1,59 @@
+import { LitElement, html, css, customElement, property } from "lit-element";
+ import { BaseButton } from "@elements/_styles/buttons";
+import {nothing} from "lit-html";
+
+export type Color = "yellow" | "pink"  ;
+export type Direction = "right" | "left"  ;
+ 
+
+@customElement("arrow-button")
+export class _ extends BaseButton {
+  static get styles() {
+    return [
+      css`
+
+      button{
+        width: 48px;
+        height: 48px;
+        cursor:pointer;
+      }
+  
+ 
+ .right{
+
+ }
+
+ .left{
+  transform: scaleX(-1);
+display:block;
+margin-left:100px;
+ }
+
+
+      `,
+    ];
+  }
+
+  @property()
+  direction:Direction   = "right";
+
+  @property()
+  color: Color  = "yellow";
+  
+  render() {
+    const { direction , color } = this;
+    const patharrow = color === "yellow" ? "chevron_arrow_yellow_right.svg"
+    : color === "pink" ? "chevron_arrow_pink.svg"
+    :color ==="blue" ? "chevron_arrow_blue_right.svg"
+     : nothing;
```

Comment:
there should never be a case of empty path, and typescript should enforce that `color` is valid. since the default `color` is yellow, the default case should similarly just be `chevron_arrow_yellow_right.svg`